Makefile

```diff
@@ -10,7 +10,7 @@ lint: vendor
 	@echo "✓ Linting source code with golangci-lint make sure you run make fmt ..."
 	@golangci-lint run --skip-dirs-use-default --timeout 5m
 
-test: lint
```

Comment:
fix all `make lint` errors and bring this line back

Reply:
Done